esp/src/src/nls/hpcc.ts

```diff
@@ -290,6 +290,7 @@ export = {
         FieldNames: "Field Names",
         File: "File",
         FileAccessCost: "File Access Cost",
+        FileCostAtRest: "File Cost At Rest",
```

Comment:
Are we concerned about a new translation string in a point release?

Reply:
I don't think so, it will just display as English until the next minor release.